metaci/release/models.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from model_utils import Choices
+from model_utils.models import StatusModel
+from model_utils.fields import AutoCreatedField, AutoLastModifiedField
+
+class Release(StatusModel):
+    STATUS = Choices('draft', 'published', 'hidden')
+    created = AutoCreatedField(_('created'))
+    modified = AutoLastModifiedField(_('modified'))
+
+    repo = models.ForeignKey('repository.Repository', 
+                            on_delete=models.CASCADE, related_name="releases")
+    
+    version_name = models.CharField(_('version name'), max_length=255, null=True, blank=True)
+    version_number = models.CharField(_('version number'), max_length=255, null=True, blank=True)
```

Comment:
What's the version name?

Reply:
They're technically two different things on a Salesforce package, though we have generally kept the our version names to be Product Name + Version Number. This could probably be an admin pre-fill feature